core/src/test/java/com/linecorp/armeria/internal/metric/MicrometerUtilTest.java

```diff
@@ -17,7 +17,7 @@
 package com.linecorp.armeria.internal.metric;
 
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
-import static org.assertj.core.api.Java6Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
Could you move this statement onto `assertThatThrownBy`?

Reply:
Thank you. Updated.